app/src/main/java/edu/uc/app/parkfinder/dao/IParkDAO.kt

```diff
@@ -6,6 +6,8 @@ import retrofit2.http.GET
 
 interface IParkDAO {
 
-    @GET("/core/country-list/data_json/data/8c458f2d15d9f2119654b29ede6e45b8/data_json.json")
+    @GET("/learn-chef/national-parks-java/master/national-parks.json")
```

Comment:
Changed GET annotation accordingly with base URL 

Once again, if you can get 
https://developer.nps.gov/api/v1/parks?parkCode=acad&api_key=rgu7ZkDUXVEl7Adf28fTyMxYe47IfOTsuUyia6ui
to work for you, it seems to have most the info you are looking for in your README

https://www.nps.gov/subjects/developer/get-started.htm
https://www.nps.gov/subjects/developer/guides.htm

these 2 links will help with use of the above API URL